-------------------------------------------------------------------------------
Test set: samples.SampleTest
-------------------------------------------------------------------------------
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.369 s <<< FAILURE! - in samples.SampleTest
second(samples.SampleTest)  Time elapsed: 0.001 s  <<< FAILURE!
java.lang.AssertionError: expected [1] but found [2]
	at samples.SampleTest.second(SampleTest.java:15)

